website/docs/developer/deployment-guide/recovery-workflow.mdx

```diff
@@ -14,18 +14,18 @@ on which components can be modified and which components should not
 be modified. Refer to the legend on the diagram for details on which
 components can be modified.
 
-Recovery processes are kicked off manually by an operator through
-the Cumulus Dashboard. The dashboard calls an API which kicks off a
-recovery workflow. Recovery is an asynchronous operation since data
-requested from GLACIER can take up to 4 hours or more to reconstitute
-in most scenarios, and DEEP_ARCHIVE can take 12 hours. 
+A developer kicks off the recovery processes manually via the UI,
```

Comment:
maybe change `via the UI` to `via the AWS UI or AWS CLI`

Reply:
updated